ambari-server/src/main/resources/Ambari-DDL-Derby-CREATE.sql

```diff
@@ -121,6 +111,16 @@ CREATE TABLE hosts (
   CONSTRAINT PK_hosts PRIMARY KEY (host_id),
   CONSTRAINT UQ_hosts_host_name UNIQUE (host_name));
 
+CREATE TABLE mpack_host_state (
```

Comment:
Any reason you moved this?

Reply:
To have the tables in the same order as in Ambari-DDL-Postgres-CREATE.sql for easier comparision.